src/mm-dialog/mm-dialog.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	scope.Dialog = Polymer({
+		is: 'mm-dialog',
+
+		iconMap: {
+			info:    { type: "info",    color: Colors.D3 },
+			success: { type: "success", color: Colors.B6 },
+			warning: { type: "warning", color: Colors.E5 },
+			error:   { type: "warning", color: Colors.C3 }
+		},
+
+		properties: {
+			type: {
+				type: String,
+				value: ''
+			},
+			header: {
+				type: String,
+				value: ''
+			},
+			hidden: {
+				reflectToAttribute: true,
+				observer: '_showHide',
+				type: Boolean,
+				value: true,
+			},
+			dismiss: {
+				observer: '_updateModalProperties',
+				type: Boolean,
+				value: true,
+			},
+			noscroll: {
+				observer: '_updateModalProperties',
+				type: Boolean,
+				value: false
+			},
+			width: {
+				observer: '_updateModalProperties',
+				type: Number,
+				value: 600
+			},
+
+			icon: {
+				type: Object,
+				computed: '_getIconAttributes(type)'
+			},
+			actions: {
+				type: Array,
+				value: function() {
+					return [{
+						label: 'OK',
+						type: 'secondary',
+						handleClick: function(e,host) {
```

Comment:
would it be a huge pain to rename this to 'callback' I feel like that is a more clear description to the end dev of what its doing...?